splunk_add_on_ucc_framework/install_python_libraries.py

```diff
@@ -55,6 +61,24 @@ def _subprocess_call(
         raise e
 
 
+def _subprocess_run(
```

Comment:
Aren't these changes are part of PR #1363?

Reply:
I have splitted this PR into two: one regarding rh_templates and one regarding splunktaucclib version. This one will be closed